datamodel/extensions/config.yaml

```
@@ -1,5 +1,9 @@
 extensions:
+  - id: demo
+    directory: demo_extension
+    variables:
+      var1: value3
+      var2: value4
```

REQ: Sinnvolle demo-extension oder gar noch expliziter `template` extension definieren.
REQ: Sinnvolle Variablen definieren & in template anwenden.
```suggestion
  - id: template
    directory: template_extension
    variables:
      myVariable: "Hello World"
      myNumber: 0
```